ztools/xssh/common.go

```diff
@@ -25,6 +25,8 @@ const (
 )
 
 // supportedCiphers specifies the supported ciphers in preference order.
+// These are *not* the supported ciphers, they are the default ciphers.
+// The all supported ciphers are listed in the ciphers.go -> cipherModes map
 var supportedCiphers = []string{
```

Comment:
This comment is still accurate. The variable is named "supportedCiphers" but does not contain all supported ciphers, it contains the default enabled supported ciphers. This is a hold-over from the original x/crypto implementation. 3des-cbc was implemented, but not included in supportedCiphers because of weakness concerns.